daemon/config_windows.go

```diff
@@ -2,14 +2,15 @@ package daemon
 
 import (
 	"os"
+	"path/filepath"
 
 	"github.com/docker/docker/api/types"
 	"github.com/spf13/pflag"
 )
 
 var (
-	defaultPidFile = os.Getenv("programdata") + string(os.PathSeparator) + "docker.pid"
-	defaultGraph   = os.Getenv("programdata") + string(os.PathSeparator) + "docker"
+	defaultPidFile string
```

Comment:
Ah, I see; to make it follow the "graphDir" location. I was confused, as on _Linux_, it doesn't follow the graphdir - is there a reason to make it follow the graphdir on Windows?